### DISCUSSION TITLE
Feature: highlight days with journals

### DISCUSSION THREAD

**Discussion by JokerQyou** [toeverything/AFFiNE#8848] [Ideas]
Currently in `Journals` there is a list of days, some of them contains content (has journal content) and some of them don't. It's not possible to tell from the day list. Maybe AFFiNE can highlight days with journals with a dot mark or something else, so that the user can see on which day he wrote a journal?

<img width="1367" alt="image" src="https://github.com/user-attachments/assets/62cdc317-4aa5-45b3-9182-3816db59289b">

**Comment by CatsJuice** [comment]
![CleanShot 2024-11-18 at 12 08 50@2x](https://github.com/user-attachments/assets/259cdc2a-959a-44e2-88ae-0b618bae03c1)

Should be able to see in sidebar calendar. BTW this feature is not stable yet!

**Reply by JokerQyou** [reply]
That's not about journals. The sidebar calendar marks days with documents (pages, journals, etc) created or updated. I was asking about Journals alone.
